File: blog-dao/src/main/java/com/entity/News.java
package com.entity;

import java.util.Date;

public class News {
    private Integer nid;

    private Date time;

    private String content;

    private String author;

    public Integer getNid() {
        return nid;
    }

    public void setNid(Integer nid) {
        this.nid = nid;
    }

    public Date getTime() {
        return time;
    }

    public void setTime(Date time) {
        this.time = time;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content == null ? null : content.trim();
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author == null ? null : author.trim();
    }
}

File: blog-dao/src/main/java/com/entity/News.kt
package com.entity

import java.util.Date

class News {
    var nid: Int? = null

    var time: Date? = null

    var content: String? = null
        set(content) {
            field = content?.trim { it <= ' ' }
        }

    var author: String? = null
        set(author) {
            field = author?.trim { it <= ' ' }
        }
}